config.go

```diff
@@ -27,6 +27,7 @@ type Config struct {
 	DiscloseIdentity bool
 	Timeline         string
 	Hooks            Hooks
+	IncludeYourself  bool
```

Comment:
Please also add a commented out instance of this option with its defualt value to config.yaml.example (with a comment line explaining what it does)

Reply:
👍